From 931d715e1c9a68cd90301b0c0659bf3a754c0001 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Mon, 11 Jun 2012 16:26:46 +0800
Subject: [PATCH 46/58] ENGR00213023 Remove vpu firmware makefile

No need make file, just do PRODUCT copy in
board configure file.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 firmware/vpu/Android.mk | 26 --------------------------
 1 file changed, 26 deletions(-)
 delete mode 100644 firmware/vpu/Android.mk

diff --git a/firmware/vpu/Android.mk b/firmware/vpu/Android.mk
deleted file mode 100644
index 7e555aa..0000000
--- a/firmware/vpu/Android.mk
+++ /dev/null
@@ -1,26 +0,0 @@
-
-ifeq ($(BOARD_HAVE_VPU),true)
-ifeq ($(EXCLUDED_CODEC_BUILD),false)
-
-LOCAL_PATH := $(call my-dir)
-
-vpu_fw_target := $(TARGET_OUT)/lib/firmware/vpu
-
-ifeq ($(BOARD_SOC_TYPE),IMX6DL)
-vpu_fw_file := vpu_fw_imx6d.bin
-else
-soc := $(shell echo "$(BOARD_SOC_TYPE)" | tr 'A-Z' 'a-z')
-vpu_fw_file := vpu_fw_$(soc).bin
-endif
-
-# Firmware
-include $(CLEAR_VARS)
-LOCAL_MODULE := $(vpu_fw_file)
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(vpu_fw_target)
-LOCAL_MODULE_TAGS := eng
-LOCAL_SRC_FILES := $(LOCAL_MODULE)
-include $(BUILD_PREBUILT)
-
-endif
-endif
-- 
1.8.0

